static/js/stream_create.js

```diff
@@ -99,9 +99,9 @@ function ajaxSubscribeForCreation(stream_name, description, principals, invite_o
                announce: JSON.stringify(announce),
         },
```

Comment:
@YJDave: Okay, I was wrong! Don't split it into two commits! But I think the commit message here should be something like `streams: Add div element indicating successful and failed stream creation.` or something! :)